core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -262,6 +262,13 @@ object Property extends Enumeration {
         "Default value is 0b meaning no limit on the size so the optimized " +
         "hash aggregation is always used.", Some("0"))
 
+  val ForceLinkPartitionsToBuckets: SQLValue[Boolean] = SQLVal[Boolean](
+    s"${Constant.PROPERTY_PREFIX}linkPartitionsToBuckets",
+    "Property to always create one bucket per partition in column/row table scans. " +
```

Comment:
@sumwale The property is not enforcing creation. It is just enforcing one partiton per bucket. Perhaps you want to re-word to -- Property to treat each bucket as one partition -- or something like that.

Reply:
Right, good catch. Should have been other way round "create one partition per bucket"